### PR TITLE
data: prashant235913

### PR DESCRIPTION
**Pull request by prashant235913**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5794"></a>

### CODE REVIEW

File: data/prashant235913.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Prashant Mishra",
+  "type": "personal",
+  "bio": "Discovering tech.Learing about Devops"
+  "avatar": "https://github.com/prashant235913.png"
```
Comment (vedprakashnautiyal) [inline data/prashant235913.json:5]:
```suggestion
```

Comment (vedprakashnautiyal) [reply data/prashant235913.json:5]: This is not required....

File: data/prashant235913.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Prashant Mishra",
+  "type": "personal",
+  "bio": "Discovering tech.Learing about Devops"
```
Comment (vedprakashnautiyal) [inline data/prashant235913.json:4]:
```suggestion
  "bio": "Discovering tech.Learing about Devops",
```

**Comment by prashant235913** [comment]
Thanku @vedprakashnautiyal  for the help
....Only ' JSON check ' is failing can you give it a look.

**Review by kumarsonsoff3 — APPROVED** [review]
Looks awesome. Great collaboration everyone. 👍

**Pull request merged**